src/main/java/mate/academy/intro/controller/BookController.java

```diff
@@ -32,4 +35,10 @@ public BookDto getBookById(@PathVariable Long id) {
     public BookDto createBook(@RequestBody CreateBookRequestDto requestDto) {
         return bookService.save(requestDto);
     }
+
+    @PutMapping("/{id}")
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    public void updateById(@PathVariable Long id) {

```

Comment:
Pass the request DTO, 
<img width="748" alt="Screenshot 2024-06-21 at 08 59 33" src="https://github.com/SerhiiAN/Spring-Boot-intro/assets/14053156/87054421-031b-4eaf-a73a-cb88d1270035">